inspectors/types/timesignaturetypes.h

```diff
@@ -0,0 +1,27 @@
+#ifndef TIMESIGNATURETYPES_H
+#define TIMESIGNATURETYPES_H
+
+#include "qobjectdefs.h"
+
+class TimeSignatureTypes
+{
+    Q_GADGET
+
+    Q_ENUMS(Mode)
```

Comment:
Please use `Q_ENUM` instead of `Q_ENUMS` as the latter is deprecated. Also, please put `Q_ENUM` after the enum type declaration.